specification/devcenter/DevCenter/examples/2023-04-01/DevBoxes_ListAllDevBoxes.json

```diff
@@ -1,9 +1,9 @@
 {
   "title": "Lists Dev Boxes that the caller has access to in the DevCenter.",
-  "operationId": "DevCenter_ListAllDevBoxes",
+  "operationId": "DevBoxes_ListAllDevBoxes",
```

Comment:
Related to a previous question, why did we change the interface for this operation?

Reply:
See note above, I think this is in the correct spot now, but if desired I can move things back